lib/rails_edge_test/dsl/edge.rb

```diff
@@ -44,6 +44,12 @@ def perform_delete(parameters={})
       process(parameters)
     end
 
+    def set_authenticity_token
```

Comment:
I ... don't really know how to test this.

Reply:
Does something like this look good?

```Ruby
it "can set authenticity token for the request" do
  test_request = nil

  Module.new do
    extend RailsEdgeTest::Dsl

    controller MyController do
      action :simple do
        edge "set the authenticity token" do
          test_request = request

          set_authenticity_token
        end
      end
    end
  end

  allow_any_instance_of(MyController).to receive(:form_authenticity_token).and_return('a_test_token')
  RailsEdgeTest::Dsl.execute!

  expect(test_request.headers['X-CSRF-Token']).to eq 'a_test_token'
end
```